earthfile2llb/earthfile2llb.go

```diff
@@ -250,9 +250,16 @@ func Earthfile2LLB(ctx context.Context, target domain.Target, opt ConvertOpt, in
 		if opt.DoSaves {
 			// Set the do saves flag, in case it was not set before.
 			sts.SetDoSaves()
+		}
+		if opt.DoPushes {
+			// Set the do saves flag, in case it was not set before.
```

Comment:
```suggestion
			// Set the do pushes flag, in case it was not set before.
```

Reply:
I reworded this comment entirely, mind taking a second look?